src/Http/include/Http/MockDownloadManagerInterface.h

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef HTTP_MOCK_DOWNLOAD_MANAGER_INTERFACE_H_
+#define HTTP_MOCK_DOWNLOAD_MANAGER_INTERFACE_H_
+
+#include <gmock/gmock.h>
+
+#include "Http/DownloadManagerInterface.h"
+
+namespace orbit_http {
+class MockDownloadManagerInterface : public DownloadManagerInterface {
```

Comment:
I think this can be called simply `MockDownloadManager`.

By the way (out of scope here), do you want the implementation of `DownloadManagerInterface` to be called `DownloadManager`? It seems that the implementation is `HttpDownloadManager`, so the interface could be called simply `DownloadManager.`